### PR TITLE
feat: secure metrics/load-test nodes from outside access

### LINKED ISSUE #54: Use a task to create the influxdb/grafana per-run info
The only remaining thing to lock-down in the security-group is the metrics instance. To fully lock it down, ardere won't be able to directly communicate with it. To create the influxdb database and grafana dashboard/datasource, we'll need to run a few commands on the metrics host directly via a task definition we run per-run instead.

### PR DESCRIPTION
**Pull request by bbangert**
Locks down the metrics node to the load-test nodes for metrics and
restricts outside grafana access to the Grafana SG.

Closes #54

### CODE REVIEW

**Comment by bbangert** [comment]
WIP so far, this appears to work, but I need to redo some tests first, and the SG is not allowing access within its own SG for an unknown reason.

File: ardere/aws.py [hunk 1]
```diff
@@ -225,6 +263,25 @@ def has_metrics_node(self, instance_type):
...
+        """Return whether the metric creation container was started"""
+        response = self._ecs_client.list_tasks(
+            cluster=self._ecs_name,
+            startedBy=self.plan_uuid
+        )
+        return bool(response["taskArns"])
+
+    def has_finished_metric_creation(self):
+        # type: () -> bool
+        """Return whether the metric creation container was started"""
```
Comment (jrconlin) [inline ardere/aws.py]: copy paste err?

**Comment by bbangert** [comment]
If the metric create script fails, then we continue anyways. Same as how we assume the shell script that's waiting for S3 is working though we don't know for sure.

File: ardere/step_functions.py [hunk 2]
```diff
@@ -403,8 +348,6 @@ def check_for_cluster_done(self):
     def cleanup_cluster(self):
         """Shutdown all ECS services and deregister all task definitions
 
-        Step 7
-
         """
```
Comment (jrconlin) [inline ardere/step_functions.py]: If you're killing steps, might want to also kill https://github.com/loads/ardere/blob/a7c985b081bc2813148ca4425e994ebddccabca9/ardere/step_functions.py#L369

File: tests/test_step_functions.py [hunk 3]
```diff
@@ -351,7 +372,9 @@ def test_validate_fail_step_name(self):
         plan['steps'][0]['name'] = '*'
         data, errors = schema.load(plan)
         eq_(errors, {'steps': {0: {'name':
-                    ['Step name contains invalid characters']}}})
+                                       [
```
Comment (jrconlin) [inline tests/test_step_functions.py]: format glitch?

**Pull request merged**